x-pack/plugins/aiops/public/components/log_categorization/format_category.tsx

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useMemo } from 'react';
+import { EuiText, EuiHorizontalRule } from '@elastic/eui';
+import { css, SerializedStyles } from '@emotion/react';
+import type { Category } from '../../../common/api/log_categorization/types';
+import { useIsDarkTheme } from '../../hooks/use_eui_theme';
+
+interface Props {
+  category: Category;
+  count?: number;
+}
+
+// copies of the syntax colors used in EuiCode
+const tokenStyleLight = css`
+  color: #765b96;
+`;
+const wildcardStyleLight = css`
```

Comment:
What do you think about dropping the `font-weight` for the wildcard style? With equal weights given to the tokens and wildcards, visually the tokens don't stand out from the variable parts of the message for me. 

This is with `font-weight: 500` for example:

<img width="508" alt="image" src="https://github.com/elastic/kibana/assets/7405507/5b141087-5605-4198-912b-391c74e56529">


<img width="570" alt="image" src="https://github.com/elastic/kibana/assets/7405507/09ace30a-e6dc-447d-a444-029c07149633">

Reply:
I'm happy to change stying here, but before I do I think it's worth discussing which parts of the text do we consider to be more important? 
The tokens, i.e. the shared words in each example, or the wildcard matches, i.e. the text which differs in each example?
I think there is an argument that the varying parts of the examples are more (or equally) interesting to the user. They'd want to see why these examples are different to each other.

Reply:
Yes good point. It was just an initial impression. I did think the tokens were more important as we're displaying patterns here, but highlighting the variable parts of the messages is also valuable. Happy to stick with this for now and see if there's any future feedback.